src/Source/DoctrineORMQuerySourceIterator.php

```diff
@@ -147,6 +158,32 @@ public function getDateTimeFormat()
         return $this->dateTimeFormat;
     }
 
+    /**
+     * @return string An ISO8601 duration
+     */
+    public function getDuration(\DateInterval $dateInterval)
```

Comment:
```suggestion
    public function getDuration(\DateInterval $interval)
```

but this is an optional change

Reply:
Made the change in latest commit